organization/_posts/2017-11-02-startup.md

```diff
@@ -0,0 +1,150 @@
+---
+title: "HSF Weekly Meeting #115, November 2, 2017"
+layout: default
+---
+
+# HSF Weekly Meeting #115, November 2, 2017
+
+#### *Present*: Pere Mato, Graeme Stewart, Eduardo Rodrigues, Benedikt Hegner, Michel Jouvin, Daniel Elvira, Andrea Valassi, David Lange, Giacomo Tenaglia, John Harvey
+
+## General News
+
+* Andrea Valassi will give a presentation on the CWP to IT department on Friday.
+
+## CWP
+
+- #### General matters and Editorial Board
+  - Graeme gave a talk on the current draft in the [Scientific Computing Forum](https://indico.cern.ch/event/663273/) 
+    - **ACTION: Add this talk to the website**
+  - Editorial Board (first) meeting on November 1: [agenda and minutes available](https://indico.cern.ch/event/677412/)
+    - Comments received from about 15 people but we know that more read it: some important ones, but no major issue identified.
+    - Decision taken to reorder the WG sections to have the more important ones first (alphabetical order made some of the most important sections to appear last, was not helping the reader to go through this quite long document). First sections will follow the experiment workflow from generators to analysis.
+    - A glossary will be added to help with acronyms and other HEP-specific vocabulary.
+    - (aggressive!) Timeline to final CWP agreed
+      - November 10: close first draft for comments
+      - November 17: release of second CWP draft
+      - December 1: close second draft for comments
+      - December 15: release of final draft
+  - We need to make a push to get people to comment on this, especially once the second draft is available.
+    - **ACTION: can Ed Board and Startup team push that please.**
```

Comment:
You may want to add a second action about "Contacting the experiments who have a computing week in the coming month (e.g. LHCb) to get a chance to present the CWP". May contribute to raise the attention of potential readers!

Reply:
I agree with that. My suggestion during the meeting was very concrete in that direction

Reply:
@graeme-a-stewart - ping. wasn't in your last update

Reply:
It was confused into the action I had below. Last commit makes this all a single unified action on the EB and ST.